src/popup/router/components/TransactionOverview.vue

```diff
@@ -0,0 +1,282 @@
+    event: { type: String },
+    contract: { type: String },
+    connect: { type: Boolean },
+    app: { type: Object },
+  },
+  computed: {
+    ...mapGetters(['account', 'getExplorerPath']),
+    transaction() {
+      if (this.connect)
+        return {
+          sender: {
+            address: this.app.host,
+            name: this.app.name,
+          },
+          recipient: this.currentAccount,
+          title: this.$t('pages.connectConfirm.connecting'),
+          desc: '...',
+        };
+      if (this.claim)
+        return {
+          sender: {
+            address: this.contract,
+            label: this.$t('transaction.overview.contract'),
+          },
+          recipient: this.currentAccount,
+          title: this.$t('transaction.overview.receiveTips'),
+          desc: this.event,
+        };
+      switch (this.txType) {
+        case TX_TYPES.txSign:
+          return {
+            sender: {
+              address: this.tx.senderId,
+              name: this.getDisplayName(this.tx.senderId),
+              label: this.$t('transaction.overview.accountAddress'),
+            },
+            recipient: {
+              address: this.tx.recipientId,
+              name: this.getDisplayName(this.tx.recipientId),
+              label: this.$t('transaction.overview.accountAddress'),
+            },
+            title: this.$t('transaction.type.spendTx'),
+            desc: this.txType,
+          };
+        case TX_TYPES.contractCall:
+          return {
+            sender: this.currentAccount,
+            recipient: {
+              address: this.tx.contractId,
+              label: this.$t('transaction.overview.contract'),
+            },
+            title: this.$t('transaction.type.contractCallTx'),
+            desc: this.txType,
+          };
+        case TX_TYPES.contractCreate:
+          return {
+            contractCreate: true,
+            sender: this.currentAccount,
+            recipient: {
+              label: this.$t('transaction.overview.contractCreate'),
+            },
+            title: this.$t('transaction.type.contractCreateTx'),
+            desc: this.txType,
+          };
+        case TX_TYPES.namePreClaim:
+        case TX_TYPES.nameClaim:
+        case TX_TYPES.nameBid:
+        case TX_TYPES.nameUpdate:
+          return {
+            aens: true,
+            sender: this.currentAccount,
+            recipient: {
+              label: this.$t('transaction.overview.aens'),
+            },
+            title: this.$t(`transaction.type`)[this.txType],
+            desc: this.txType,
+          };
+        default:
+          throw new Error('Unsupported transaction type');
+      }
+    },
+    txType() {
+      return TX_TYPES_MDW[this.tx.type] || OBJECT_ID_TX_TYPE[this.tx.tag];
+    },
+    currentAccount() {
+      return {
+        address: this.account.address,
+        name: this.account.name,
+        label: this.$t('transaction.overview.accountAddress'),
+      };
+    },
```

Comment:
proposing to modify the `account` getter instead:
```js
...mapState({
  account(state, { account }) {
    return {
      ...account,
      label: this.$t('transaction.overview.accountAddress'),
    };
  },
}),
```
then we won't have `account` and `currentAccount` that looks different but means almost the same